core-skills/web-application-development-with-react/README.md

```diff
@@ -126,6 +128,8 @@ The candidate should create a React application which satisfies the following.
     the form data.
   - Candidate should walk assessor through tests which give maintainers confidence 
     that the component submits the data from the form to the API.
+  - Testing this component should include unit tests that demonstrate the boundary between the view logic and domain logic,
```

Comment:
dupe?

Reply:
Two components, two tests, repeated the requirement to make it clear it was needed in both cases.

Reply:
hmm ok :thinking: